src/v/cluster/controller_backend.cc

```diff
@@ -715,7 +715,7 @@ ss::future<> controller_backend::reconcile_ntp(deltas_t& deltas) {
             continue;
         } catch (...) {
             vlog(
-              clusterlog.error,
+              clusterlog.warn,
```

Comment:
I'm okay with this too, but did you consider catching the filesystem error and warning just on that? I'm not sure what other exceptions are lurking here, so maybe this is actually fine.

Reply:
the one i saw are metrics double registration, probably bad_alloc